packages/pwa/app/pages/product-list/index.jsx

```diff
@@ -390,6 +396,14 @@ const ProductList = (props) => {
                                                           : removeItemFromWishlist
                                                       return action(productSearchItem)
                                                   }}
+                                                  imageProps={{
+                                                      sizes: {
+                                                          base: 'calc(100vw / 2)',
+                                                          md: `calc((100vw - ${filtersContainerWidth}) / 3)`,
```

Comment:
Your sizes don't have to be 100% accurate. Otherwise, the sizes would get complicated if you choose to measure the margins/paddings too. In this case, for `md` breakpoint, the width of a product tile would be roughly calculated as:
1. 100% of viewport width minus the filters container → you get the width of the main content
2. Then you divide it up into 3 columns